extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/topage/ToInstantPageFromInt96.java

```diff
@@ -91,11 +93,15 @@ public final long[] convertResult(@NotNull final Object result) {
         final long[] resultLongs = new long[resultLength];
 
         for (int ri = 0; ri < resultLength; ++ri) {
-            final ByteBuffer resultBuffer = ByteBuffer.wrap(results[ri].getBytesUnsafe());
-            resultBuffer.order(java.nio.ByteOrder.LITTLE_ENDIAN);
-            final long nanos = resultBuffer.getLong();
-            final int julianDate = resultBuffer.getInt();
-            resultLongs[ri] = (julianDate - JULIAN_OFFSET_TO_UNIX_EPOCH_DAYS) * (NANOS_PER_DAY) + nanos + offset;
+            if (results[ri] != null) {
```

Comment:
The code is clear enough as written, but I'd like to suggest inside the for loop you do:

```
[...]

if (results[ri] == null) {
    resultLongs[ri] = NULL_LONG;
    continue;
}
// The rest of the code for the results[ri] != null case above go here.
```

Done that way, you avoid the additional indentation on the (longer) block of code for the non-null case.
This is a general idea btw; when you can quickly filter out cases with `return` or `continue` (not always possible),
doing
```
if (condition) {
    // handle true case, easy and short
} else {
    // handle false case, long and complicated
}
```
is harder to read / slightly worse than
```
if (condition) {
    // handle true case, easy and short
    return;
}
// handle false case, long and complicated
```

Of course the compiler won't mind either way, but as Abelson said once “Programs must be written for people to read, and only incidentally for machines to execute.”